src/components/AddButton/AddButton.vue

```diff
@@ -1,9 +1,17 @@
 <template>
-    <li class="addButton">
-        <a @click="addIcon('toto')">
-          <span class="addButton__icon"></span>
+    <li class="addButton icon-list__item">
+        <router-link class="addButton__link"
+          to="/upload">
+          <span class="addButton__icon-container">
+            <svg xmlns="http://www.w3.org/2000/svg"
```

Comment:
you can call svg throug background img ? webpack will do great job for you.

Reply:
I've used the <use> tag instead : so we can still style our icons the way we want.